api-test/src/main/resources/pms/GetAllDeviceTypes/GetAllDeviceTypes.yml

```
@@ -38,7 +38,7 @@ GetAllDeviceTypes:
     
       partner_DeviceTypeFilter_All_Valid_Smoke:
       endPoint: /v1/partnermanager/devicedetail/deviceType/filtervalues
```

YAMLlint (1.38.0)

[error] duplication of key "endPoint" in mapping

(key-duplicates)